tools/vp20compiler/Makefile

```diff
@@ -8,7 +8,7 @@ SRCS = nvvertparse.c \
 
 OBJS = $(SRCS:.c=.o)
 
-CFLAGS = -std=c99
+CFLAGS = -std=gnu99
```

Comment:
just FYI: I split this into a seperate commit in my WIP branches "vp20compiler: Use gnu99 for strdup support"